tools.js

```diff
@@ -587,24 +585,11 @@ module.exports = (() => {
 					// don't revalidate what we already validate
 					if (banlistTable[toId(subformat.banlist[i])]) continue;
 
-					banlistTable[subformat.banlist[i]] = subformat.name || true;
 					banlistTable[toId(subformat.banlist[i])] = subformat.name || true;
-
-					let complexList;
 					if (subformat.banlist[i].includes('+')) {
-						if (subformat.banlist[i].includes('++')) {
-							complexList = subformat.banlist[i].split('++');
-							for (let j = 0; j < complexList.length; j++) {
-								complexList[j] = toId(complexList[j]);
-							}
-							format.teamBanTable.push(complexList);
-						} else {
-							complexList = subformat.banlist[i].split('+');
-							for (let j = 0; j < complexList.length; j++) {
-								complexList[j] = toId(complexList[j]);
-							}
-							format.setBanTable.push(complexList);
-						}
+						banlistTable[subformat.banlist[i]] = subformat.banlist[i].split(/\++/).map(toId);
```

Comment:
I thought it would help to make it possible at some point in the future for a format to use a nearly-standard ruleset but without some of the child rules (e.g. Itemize would want an OU ruleset but without the Soul Dew ban); I couldn't come up with a clean solution when there were multiple tables.